src/_version_check.sh

```diff
@@ -103,19 +103,18 @@ function version_check() {
         debug "Looking for Running version $running_ver or Breaking version $breaking_ver"
         while [ $page -lt 600 ]
           do
+            upstream_tags+=(`wget --header="Authorization: JWT "${JWT_TOKEN} -O - "https://hub.docker.com/v2/repositories/plextrac/plextracapi/tags/?page=$page&page_size=1000" -q | jq -r .results[].name | grep -E '(^[0-9]\.[0-9]*$)' || true`)
```

Comment:
This was moved up and should haven't a problem with the validation of the version. Helps populate the version before the if check happens